docs/docsite/rst/scenario_guides/guide_gce.rst

```diff
@@ -104,120 +111,123 @@ Set the following environment variables before running Ansible in order to confi
-
-Now let's see if we can use the inventory script to talk to Google.
+The best way to interact with your hosts is to use the gcp_compute inventory plugin, which dynamically queries GCE and tells Ansible what nodes can be managed.
 
-.. code-block:: bash
-
-    $ GCE_INI_PATH=~/.gce.ini ansible all -i gce.py -m setup
-    hostname | success >> {
-      "ansible_facts": {
-        "ansible_all_ipv4_addresses": [
-          "x.x.x.x"
-        ],
-
-As with all dynamic inventory scripts in Ansible, you can configure the inventory path in ansible.cfg.  The recommended way to use the inventory is to create an ``inventory`` directory, and place both the ``gce.py`` script and a file containing ``localhost`` in it.  This can allow for cloud inventory to be used alongside local inventory (such as a physical datacenter) or machines running in different providers.
-
-Executing ``ansible`` or ``ansible-playbook`` and specifying the ``inventory`` directory instead of an individual file will cause ansible to evaluate each file in that directory for inventory.
-
-Let's once again use our inventory script to see if it can talk to Google Cloud:
-
-.. code-block:: bash
-
-    $ ansible all -i inventory/ -m setup
-    hostname | success >> {
-      "ansible_facts": {
-        "ansible_all_ipv4_addresses": [
-            "x.x.x.x"
-        ],
-
-The output should be similar to the previous command.  If you're wanting less output and just want to check for SSH connectivity, use "-m" ping instead.
-
-Use Cases
----------
-
-For the following use case, let's use this small shell script as a wrapper.
-
-.. code-block:: bash
-
-  #!/usr/bin/env bash
-  PLAYBOOK="$1"
+To use the gcp_compute inventory plugin, create a file that ends in .gcp.yml file in your root directory. The gcp_compute inventory script takes in the same authentication
+information as any module.
 
-  if [[ -z $PLAYBOOK ]]; then
-    echo "You need to pass a playbook as argument to this script."
-    exit 1
-  fi
+Here's an example of a valid inventory file:
 
-  export SSL_CERT_FILE=$(pwd)/cacert.pem
-  export ANSIBLE_HOST_KEY_CHECKING=False
+.. code-block:: yaml
 
-  if [[ ! -f "$SSL_CERT_FILE" ]]; then
-    curl -O http://curl.haxx.se/ca/cacert.pem
-  fi
+    plugin: gcp_compute
+    projects:
+      - google.com:graphite-playground
+    filters:
+    auth_kind: serviceaccount
+    service_account_file: /home/alexstephen/my_account.json
 
-  ansible-playbook -v -i inventory/ "$PLAYBOOK"
 
+Executing ``ansible-inventory --list -i <filename>.gcp.yml`` will create a list of GCP instances that are ready to be configured using ANsible.
 
 Create an instance
 ``````````````````
 
-The GCE module provides the ability to provision instances within Google Compute Engine. The provisioning task is typically performed from your Ansible control server against Google Cloud's API.
+The full range of GCP modules provide the ability to create a wide-variety of
```

Comment:
doesn't need a hyphen (wide variety, not wide-variety)

Reply:
Done.